### DISCUSSION TITLE
[Firebase Authentication] Github custom domain OAuth not working on Android

### DISCUSSION THREAD

**Discussion by khang-nd** [capawesome-team/capacitor-firebase#420] [Q&A] [answered]
I managed to set up a custom domain (`https://auth.xxx.com/__/auth/handler`) for Firebase Authentication in my project, and it works fine on web since I modified the `authDomain` in the firebase config.

However, on Android, it appears the default domain (`https://xxx.firebaseapp.com__/auth/handler`) is still invoked initially and only after the authentication process, it redirects to the custom domain, which then effectively causes an issue that the auth token does not persist on the custom domain and the Firebase cannot login the users. And as the Android version of Firebase Auth relies on a `google-services.json`, which doesn't expose the `authDomain` field to be configured, so I'm totally blocked.

Does anyone have the same problem? Did you manage to resolve this issue? Thank you for reading.

**Comment by robingenz** [comment, marked as answer]
This does not seem to be supported by Firebase at the moment, see https://github.com/firebase/firebase-android-sdk/issues/4805#issuecomment-1487268359.
